ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-09.xml

```diff
@@ -328,8 +332,8 @@
 
 			</section> 
 
-			<section title="Safety guidelines">
-				<t>If the client accept the extension, the server could increase the initial congestion window. The maximum number of initial data packets that can be sent without acknowledgment needs to be chosen to avoid congestion collapse. A mechanism is needed to ensure that the information sent by the server is relevant and does not harm.</t>
+			<section anchor="safety_guide" title="Safety guidelines">
+				<t>If the client accept the extension, the server could increase the initial congestion window. The maximum number of initial data packets that can be sent without acknowledgment needs to be chosen to avoid congestion collapse. A mechanism is needed to ensure that the information sent by the server is relevant and does not harm. The proposed mechanisms would be limited by other rate-limitation mechanisms of QUIC, such as the address validation process that is used to prevent amplification attacks <xref target="I-D.ietf-quic-transport"></xref>.</t>
```

Comment:
```suggestion
<t>If the client accepts the extension, the server could increase the initial congestion window. The maximum number of packets that can be sent without acknowledgment needs to be chosen to avoid congestion collapse. Additionally, a mechanism is needed to ensure that the information sent by the server is relevant and does not harm. Finally, the proposed mechanisms SHOULD be limited by other rate-limitation mechanisms of QUIC, such as the address validation process that is used to prevent amplification attacks <xref target="I-D.ietf-quic-transport"></xref>.</t>
```

"initial data" has a specific meaning in QUIC (initial encryption level). If not removed, this should probably be "early data packets" or "0-RTT data reply packets" or similar. 

I'm also not sure what this means concretely:
> Additionally, a mechanism is needed to ensure that the information sent by the server is relevant and does not harm

Reply:
Indeed, using initial data may be confusing - thank you very much for spotting this! 
Instead of 'does not harm', how about 'does not add congestion to a congested network' ?
Or 'Additionnaly, safety guidelines are necessary to ensure that whatever the information sent by the server, the usage of the extension does not add congestion to a congested network' ?

Reply:
If that's the intent, then I struggle to see how it's very different from the first sentence: 
> The maximum number of packets that can be sent without acknowledgment needs to be chosen to avoid congestion collapse

Of course there are nuances between full congestion collapse and "adds congestion", but I still think both can then be joined in a single point.

Then you have two main points:
- Don't create/increase congestion
- Don't be an amplification attack vector

Reply:
Text proposed in commit https://github.com/NicoKos/QUIC_HIGH_BDP/commit/83d4bd18a0d4cc409534391fd32ec079981fda90